utils/SnapPy/Snappy/EEMEP/resources/job_script_ppi_centos7_direct_npp.job

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 ## {runtag} {rundir} {year}-{month}-{day}T{hour} {runhour}
 
-#$ -N eemep_nuc
+#$ -N eemep
```

Comment:
Is this name used by the script to locate the right job?

Reply:
I think(!) it the name given to the eemep run on PPI

Reply:
This name is only used by humans during `squeue`. Both eemep and eemep_nuc are fine with me.